src/searchlog.c

```diff
@@ -150,23 +148,63 @@ void displayCallInfo(dxcc_data *dx, int z, char *pxstr) {
     }
 }
 
+
+#define PARTIALS_ROWS   5
+#define PARTIALS_COLS   40      // including margins (1+1 columns)
+#define PARTIALS_X0     1
+#define PARTIALS_Y0     0
+
+//
+// return: true if display is full
+//
+static int show_partial(int *row, int *col, char *call,
+			GHashTable *callset,
+			int *nr_suggested, char *suggested_call) {
+
+    if (!g_hash_table_add(callset, call)) {
+	return 0;   // already shown
+    }
+
+    mvprintw(PARTIALS_X0 + *row, PARTIALS_Y0 + *col, "%s ", call);
```

Comment:
Please recheck. X normally is in column direction, Y are row positions.

Reply:
corrected